aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -205,10 +205,17 @@ def _get_dims_and_coords_from_ome(
         # If non global linear timescale, we need to create an array of every plane
         # time value
         elif scene_meta.pixels.size_t > 1:
-            t_index_to_delta_map = {
-                p.the_t: p.delta_t for p in scene_meta.pixels.planes
-            }
-            coords[DimensionNames.Time] = list(t_index_to_delta_map.values())
+            if len(scene_meta.pixels.planes) > 0:
+                t_index_to_delta_map = {
+                    p.the_t: p.delta_t for p in scene_meta.pixels.planes
+                }
+                coords[DimensionNames.Time] = list(t_index_to_delta_map.values())
+            else:
+                coords[DimensionNames.Time] = np.linspace(
+                    0,
+                    scene_meta.pixels.size_t,
+                    scene_meta.pixels.size_t,
+                )
```

Comment:
Not sure I understand.  This is creating a list of values [0,1,2,... size_t-1].  Is that what you want the comment to say?

Reply:
You made me look closer and it looks like I need to set endpoint=False here, actually

Reply:
Oh woops. My bad, I was thinking of the `np.arange` API. `linspace` is correct here. I think this is accurate? Xarray would throw an error if you provided a T dimension with size different than the attached coordinate plane.

Reply:
!!! I know why I thought this was wrong. It's because I was wrong on the other uses of linspace. See line 200, the if block right above this. That is wrong I think.

Reply:
I think the if block above this should be `np.arange(0, scene_meta.pixels.size_t, scene_meta.pixels.time_increment_quantity)`

Reply:
Want me to put that in this PR?

Reply:
Yes please.